src/App/Components/DisplayArticle/Ratings.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable react/no-unused-state */
```

Comment:
@nkpremices why do you have an unused state? you should fix the issue, not disable the rule.

Reply:
Oh! I just forgot to remove it @MemunaHaruna

Reply:
lemme do it